Rakefile

```diff
@@ -10,6 +10,9 @@ $:.unshift("#{$GEMS_DIR}/aws-sdk-core/lib")
 $:.unshift("#{$GEMS_DIR}/aws-partitions/lib")
 $:.unshift("#{$GEMS_DIR}/aws-eventstream/lib")
 $:.unshift("#{$GEMS_DIR}/aws-sigv4/lib")
+Dir.glob("#{$GEMS_DIR}/aws-sdk-*/lib").each do |lib|
+  $:.unshift(lib)
+end
```

Comment:
It is used to know the exact arguments of `Client#initialize`.
Is it acceptable?